apps/api/src/config/typeorm/configuration.ts

```diff
@@ -16,6 +16,6 @@ export default new DataSource({
   entities: [],
   autoLoadEntities: true,
   migrations: ['src/database/migrations/**'],
-  migrationsTableName: 'jobs',
+  migrationsTableName: 'jobdetails',
```

Comment:
No, this is a migration file. So it should be `job_migrations` or just `migrations`. Also I though  we named the entity as Summary not Job?

Reply:
Okay, it was earlier decided to keep tablenames as jobs and files. will change it to summary and migrations table name to migrations.